tests/test_categorical.py

```diff
@@ -86,6 +86,58 @@ def test(self):
 ]
 
 
+class ReplaceCategoricalColumnsTest(unittest.TestCase):
+    def test_use(self):
+
+        expected = [
+            # Seen values
+            {"key": 1, "column1": 10, "column2": 2, "column3": None},
+            # Unseen values
+            {"key": 2, "column1": None, "column2": None, "column3": None},
+            # Mixed
+            {"key": 3, "column1": None, "column2": 3, "column3": None},
+            # Repeat Row
+            {"key": 4, "column1": None, "column2": 3, "column3": None},
```

Comment:
What is the intention of testing repeated rows?

Reply:
Testing repeat substitutions give the same result for the same values.